cmd/containerd-shim-runhcs-v1/options/next.pb.txt

```diff
@@ -1,193 +0,0 @@
-file {
```

Comment:
I think you keep deleting this because you are running protobuild from the root. Instead run it two times once for `protobuild ./cmd/containerd-shim-runhcs-v1/options` and another time for `protobuild ./cmd/containerd-shim-runhcs-v1/stats`. Maybe its time for a makefile :)

Reply:
That still causes the file to be deleted